### PR TITLE
add Checkbox

### PR DESCRIPTION
**Pull request by alicie232**
Each task has been given its own checkbox. The tasks can be checked off individually and are then crossed out. Currently the status is not saved yet. This function will come next :)

### CODE REVIEW

**Comment by chris-yooo** [comment]
Hab mir die letzten commits angeschaut sieht doch geil aus :) clean dermaßen ausgelagert alles -> MERGE du selbst nä ^^

File: src/components/DailyTasks.js [hunk 1]
```diff
@@ -26,7 +15,7 @@ export default function DailyTasks() {
         </p>
       </header>
       <Wrapper>
-        <TaskList tasks={dailyTasks} />
+        <TaskList todos={todos} setTodos={setTodos} />
```
Comment (stefankorder) [inline src/components/DailyTasks.js:18]: You should not pass down the set function, instead you should create a function that is passed down and call the set function in it.

File: src/components/TaskList.js [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
     <ul>
-      {tasks.map(task => (
-        <li key={task}>{task}</li>
+      {todos.map(todo => (
+        <li key={todo.id} style={{listStyle: 'none'}}>
+          <input
+            type="checkbox"
+            checked={todo.isChecked}
+            onChange={() => {
+              setTodos(todos.map(todo_ => (todo_.id === todo.id ? {...todo_, isChecked: !todo.isChecked} : todo_)));
```
Comment (stefankorder) [inline src/components/TaskList.js:10]: How i wrote in the first comment.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! There are 1 hint you should take in progress.

**Review by stefankorder — APPROVED** [review]
Good work! Maybe you could take the two hints we talked about in action.

File: src/components/DailyTasks.js [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
 
 export default function DailyTasks() {
+  const [todos, setTodos] = useState(dailyTasks);
+
+  function handleTodos(todoToHandle) {
+    setTodos(
+      todos.map(todo => {
+        return todo.id === todoToHandle.id ? {...todo, isChecked: !todo.isChecked} : todo;
+      })
+    );
```
Comment (actyralf) [inline src/components/DailyTasks.js:14]: Could be even shorter as your function consists of only one line returning something:

`setTodos(todos.map(todo => (todo.id === todoToHandle.id ? {...todo, isChecked: !todo.isChecked} : todo)));`

In this case you can omit the curly braces AND the return keyword and just write the expression you want to return. However, there is nothing wrong with your code and in the beginning it's sometimes easier to read if you have an explicit return.

**Review by actyralf — APPROVED** [review]
Well done!

**Pull request merged**